docs/7.x/pack.md

```diff
@@ -905,6 +905,69 @@ installer:
 !!! note
 	Currently, only one setup endpoint per application is supported.
 
+## Licensing (Enterprise)
+
+Gravity Enterprise provides the ability to generate licenses and require a license for cluster installations.  Licenses are restrictable for the number of nodes per cluster and a license expiration date.  You have the option of requiring the application to stop when a license expires.
+
+Here are the commands available in `gravity` for creating, installing, updating and displaying the license in the cluster.  
+
+| Command        | Description |
+|----------------|-------------|
+| `gravity license new`   | Generates a new license
+| `gravity license install`    | Install (or update) a cluster license.
+| `gravity license show`    | Downloads a Cluster Image from a Gravity Hub.
+To see all the options on each command use `--help` as a paramter on each command (Ex: `gravity license new --help`)
```

Comment:
parameter
To see usage details for each command, append `--help` (Ex. `gravity license new --help`)